app/io/pathfinder/models/Commodity.scala

```diff
@@ -5,11 +5,12 @@ import javax.persistence.{Column, Entity, GeneratedValue, GenerationType, Id, Ma
 import com.avaje.ebean.Model
 import com.fasterxml.jackson.annotation.JsonIgnore
 import io.pathfinder.data.{Resource, EbeanCrudDao}
+import io.pathfinder.websockets.WebSocketDao
 import play.api.libs.json.{Json, Format}
 
 object Commodity {
     val finder: Model.Find[Long,Commodity] = new Model.Finder[Long,Commodity](classOf[Commodity])
-    object Dao extends EbeanCrudDao[Long,Commodity](finder)
+    object Dao extends WebSocketDao[Commodity](finder)
```

Comment:
It is for push requests, the websocket dao is intended to push notifications to subscribed websockets. Anyone using the dao to access its data doesn't need to worry about that though